src/main/webapp/app/tutor-course-dashboard/tutor-participation-graph/tutor-participation-graph.component.html

```diff
@@ -12,23 +12,40 @@
         >
             <fa-icon [icon]="'chalkboard-teacher'"></fa-icon>
         </li>
-
         <li class="progress-bar-li" [ngClass]="calculateClassProgressBar()">
             <div class="progress position-relative" [ngbTooltip]="'artemisApp.tutorCourseDashboard.assessStudentsSubmissions' | translate">
                 <div
                     class="progress-bar"
                     role="progressbar"
-                    [ngClass]="calculateProgressBarClass()"
+                    [ngClass]="calculateProgressBarClass(percentageAssessmentProgress)"
                     [style.width]="percentageAssessmentProgress + '%'"
                     attr.aria-valuenow="{{ percentageAssessmentProgress }}"
                     aria-valuemin="0"
                     aria-valuemax="100"
                 >
-                    <span class="justify-content-center d-flex position-absolute w-100" [ngClass]="chooseProgressBarTextColor()">
+                    <span class="justify-content-center d-flex position-absolute w-100" [ngClass]="chooseProgressBarTextColor(percentageAssessmentProgress)">
                         {{ numberOfAssessments }} / {{ numberOfParticipations }} ({{ percentageAssessmentProgress + '%' }})
                     </span>
                 </div>
             </div>
         </li>
+        <li class="progress-bar-li" [ngClass]="calculateClassProgressBar()">
+            <div class="progress position-relative" [ngbTooltip]="'artemisApp.tutorCourseDashboard.evaluateStudentsComplaints' | translate">
+                <div
+                    class="progress-bar"
+                    role="progressbar"
+                    [ngClass]="calculateProgressBarClass(percentageComplaintsProgress)"
+                    [style.width]="percentageComplaintsProgress + '%'"
+                    attr.aria-valuenow="{{ percentageComplaintsProgress }}"
+                    aria-valuemin="0"
+                    aria-valuemax="100"
+                >
+                    <span class="justify-content-center d-flex position-absolute w-100" [ngClass]="chooseProgressBarTextColor(percentageComplaintsProgress)">
+                        {{ this.numberOfComplaints - this.numberOfOpenComplaints + (this.numberOfMoreFeedbackRequests - this.numberOfOpenMoreFeedbackRequests) }} /
+                        {{ numberOfComplaints + numberOfMoreFeedbackRequests }} ({{ percentageComplaintsProgress + '%' }})
+                    </span>
+                </div>
+            </div>
```

Comment:
Since this is basically the same code as above, we can refactor this into a small subcomponent.